tests/lib/nested.sh

```diff
@@ -1155,6 +1154,16 @@ nested_force_stop_vm() {
     systemctl stop "$NESTED_VM"
 }
 
+nested_ensure_ovmf() {
+    if [ -d "${NESTED_ASSETS_DIR}/ovmf" ]; then
+        return
+    fi
+    if ! [ -f "${NESTED_ASSETS_DIR}/ovmf.snap" ]; then
+        (cd "${TESTSLIB}/snaps/ovmf"; run_snapcraft --use-lxd --verbosity quiet --output="${NESTED_ASSETS_DIR}/ovmf.snap")
```

Comment:
I think I'd rather download the snap from store rather than build it, or even fail explicitly and expect `run-spread` to do the download.

Reply:
I agree. I will keep it in the draft for now. When the tests seem to work, I will add some CI step that uploads it.